Another_notebook.ipynb

```diff
@@ -14,7 +14,7 @@
   },
```

Comment:
<p>This figure has been update and is shown here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/agamemnonc/reviewnb-test/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>